### PR TITLE
Add Italian subtitles for the what-is-monero video in home

### PR DESCRIPTION
**Pull request by staff91**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -16,6 +16,8 @@
                             {% else %}
                                 <source src="/media/{{ site.lang }}/Monero_Promo.m4v">
                             {% endif %}
+                            <track label="English" kind="subtitles" srclang="en" src="/media/subtitles/en/monero-promo.vtt" default>
```
Comment (erciccione) [inline index.html:19]: This will create conflicts and differs from #1373. Would be better to base your PR on that one.

**Review by erciccione — CHANGES_REQUESTED** [review]

File: media/subtitles/it/monero-promo.vtt [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+00:00:00.400 --> 00:00:06.320
+Per la maggior parte delle persone, la privacy finanziaria è molto importante, ma le banche, le grandi società e
+
+00:00:06.320 --> 00:00:11.520
+persino i governi stanno vedendo i loro documenti
+compromessi. La conclusione innegabile è che non
+
+00:00:11.520 --> 00:00:18.640
+esiste un luogo sicuro dove condurre transazioni private. Beh, non ne esisteva uno fino ad ora. Ed ecco a voi Monero.
```
Comment (erciccione) [inline media/subtitles/it/monero-promo.vtt:13]:
```suggestion
esiste un luogo sicuro dove condurre transazioni private. Beh, non ne esisteva uno fino ad ora. Ecco a voi Monero.
```

File: media/subtitles/it/monero-promo.vtt [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+00:00:18.640 --> 00:00:25.120
+Monero è una valuta sicura, privata, non rintracciabile. Con Monero sei tu la tua banca.
```
Comment (erciccione) [inline media/subtitles/it/monero-promo.vtt:16]:
```suggestion
Monero è una valuta sicura, privata, non tracciabile. Con Monero sei tu la tua banca.
```

File: media/subtitles/it/monero-promo.vtt [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+00:00:25.840 --> 00:00:32.560
+Solo tu controlli e sei responsabile dei tuoi
+fondi. Monero è privato, questo significa che le aziende
+
+00:00:32.560 --> 00:00:38.160
+possono tenere segreti i loro fornitori,
+mentre i cittadini sfuggono alle repressioni del governo
+
+00:00:38.160 --> 00:00:44.160
+e ai vicini ficcanaso o ai truffatori. Monero può anche essere selettivamente trasparente e puoi scegliere
```
Comment (erciccione) [inline media/subtitles/it/monero-promo.vtt:27]:
```suggestion
possono tenere segreti i loro fornitori,
e individui possono sfuggire a governi repressivi

00:00:38.160 --> 00:00:44.160
e a vicini ficcanaso o ai truffatori. Monero può anche essere selettivamente trasparente e puoi scegliere
```

File: media/subtitles/it/monero-promo.vtt [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+00:00:44.160 --> 00:00:49.920
+chi vuoi che veda le transazioni. Monero è scalabile dinamicamente, il che lo rende molto diverso
```
Comment (erciccione) [inline media/subtitles/it/monero-promo.vtt:30]:
```suggestion
chi vuoi che veda le transazioni. Monero scala dinamicamente, il che lo rende molto diverso
```

Not sure about this one, but "e' scalabile" doesn't sound right.

File: media/subtitles/it/monero-promo.vtt [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+00:00:49.920 --> 00:00:56.880
+da un Bitcoin. Inoltre, la ricompensa del blocco non scenderà mai al di sotto di 0,3 XMR e l'inflazione nominale
```
Comment (erciccione) [inline media/subtitles/it/monero-promo.vtt:33]:
```suggestion
da Bitcoin. Inoltre, la ricompensa di blocco non scenderà mai al di sotto di 0,3 XMR e l'inflazione nominale
```

See the [terminology guide for Italian](https://github.com/monero-ecosystem/monero-translations/blob/master/terminology-guides/italian-terminology.md)

File: media/subtitles/it/monero-promo.vtt [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+
+00:00:56.880 --> 00:01:03.440
+rimarrà a 0,3 XMR al minuto. Monero è decentralizzato, ha una comunità di sviluppo attivo con più
+
+00:01:03.440 --> 00:01:08.880
+di 30 collaboratori. Riprendi il controllo delle
```
Comment (erciccione) [inline media/subtitles/it/monero-promo.vtt:39]:
```suggestion
di 30 sviluppatori. Riprendi il controllo delle
```

**Review by erciccione — COMMENTED** [review]
Review of the Italian translation of the subtitles

**Comment by staff91** [comment]
@erciccione Thanks for review!

**Comment by erciccione** [comment]
@staff91 please implement the suggested changes and rebase :)

**Comment by erciccione** [comment]
ping @staff91

**Comment by erciccione** [comment]
This PR is abandoned. Recovered in #1450.

**Pull request closed without merging**